language-server/modules/langserver-core/src/test/resources/completion/let_expression_context/source/let_expr_ctx_source17.bal

```diff
@@ -0,0 +1,3 @@
+public function main() {
+    string:Char ch = string:
```

Comment:
missing let keyword

Reply:
Fixed in [c9dbc2b](https://github.com/ballerina-platform/ballerina-lang/pull/39994/commits/c9dbc2b76270bb9e5f5f9dbb930c2ce8e9a17b46)